pyccel/stdlib/cwrapper/CMakeLists.txt

```diff
@@ -0,0 +1,13 @@
+
+
+add_library(cwrapper OBJECT cwrapper.c)
```

Comment:
I have noticed that these CMake and Meson files start with one or two blank lines. Is there a reason for that?